scripts/source/process-cnp-docs.sh

```diff
@@ -25,4 +25,5 @@ node $DESTINATION_CHECKOUT/scripts/fileProcessor/main.mjs \
   -p "cnp/add-frontmatters" \
   -p "cnp/rename-to-mdx"
 
-cp -rf src/* $DESTINATION_CHECKOUT/advocacy_docs/kubernetes/cloud_native_postgresql/
+rsync -a --delete src/ $DESTINATION_CHECKOUT/advocacy_docs/kubernetes/cloud_native_postgresql/
+cp $DESTINATION_CHECKOUT/merge_sources/kubernetes/cloud_native_postgresql/interactive_demo.mdx $DESTINATION_CHECKOUT/advocacy_docs/kubernetes/cloud_native_postgresql/interactive_demo.mdx
```

Comment:
This is copying the `interactive_demo.mdx` from the merge_sources dir. It doesn't exist in the CNP repo, but the processor adds a reference to it in `quickstart.mdx`, and in `index.mdx`.

IIRC, the reason I wasn't deleting files previously was because I was trying to preserve the quickstart file, and later found it in merge_sources.